swagger.yaml

```diff
@@ -119,6 +119,29 @@ paths:
           $ref: '#/responses/InternalError'
 
   /jobs/{id}/tasks:
+    get:
+      tags:
+        - "Private User"
+      summary: Get a list of tasks for a given job.
+      description: "Get a list of tasks, for a given search reindex job, which meet the criteria specified by the four query parameters."
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: "#/parameters/limit"
+        - $ref: "#/parameters/offset"
+        - $ref: "#/parameters/sort"
```

Comment:
I wouldn't add sort parameter to the list endpoints in this application yet because we are not allowing API users to set sort in the query to affect the order of results.

Also the sort field is not returned in the response body currently, so its worth removing it from the swagger spec

Reply:
good point - done